lms/static/scripts/frontend_apps/components/test/FilePickerApp-test.js

```diff
@@ -408,43 +408,46 @@ describe('FilePickerApp', () => {
       const wrapper = renderFilePicker();
       const description = wrapper.find('[data-testid="content-summary"]');
       assert.equal(description.text(), 'https://test.com/example.pdf');
-      assert.isTrue(wrapper.exists('Button[data-testid="edit-content"]'));
+      assert.isTrue(wrapper.exists('button[data-testid="edit-content"]'));
     });
 
     it('shows "Save" button instead of "Continue" button', () => {
       const wrapper = renderFilePicker();
-      const saveButton = wrapper.find('Button[data-testid="save-button"]');
+      const saveButton = wrapper.find('button[data-testid="save-button"]');
       assert.equal(saveButton.text(), 'Save');
     });
 
-    it('allows editing content selection', () => {
-      const wrapper = renderFilePicker();
+    [true, false].forEach(groupsEnabled => {
```

Comment:
I parameterized this test because I almost introduced a logic bug (yikes!) that this test previously didn't catch.